.gitignore

```diff
@@ -58,3 +58,4 @@ Thumbs.db
 *~
 
 /**/node_modules/
+node/
```

Comment:
AFAIK there is no node folder generated, so why is it added ?

Reply:
If you run the build you'll see that a node folder IS generated. That's where the frontend plugin puts the local node installation. :D